backend/question_service/controllers/questionController.js

```diff
@@ -181,10 +182,55 @@ const deleteQuestion = async (req, res) => {
   }
 };
 
+// get questions from an array of question ids
+const getQuestions = async (req, res) => {
+  try {
+    const questionIds = req.body.questionIds;
+
+    if (!questionIds) {
+      return res.status(400).json({
+        message: "Question IDs is (are) required",
+      });
+    }
+
+    const questionDataArray = []; // push each qn data into here
+
+    for (const questionId of questionIds) {
+      const questionDoc = await db
+        .collection("questions")
+        .doc(questionId)
+        .get();
+
+      const questionData = questionDoc.data();
+
+      if (!questionData) {
+        console.log("Question doesn't exists!");
```

Comment:
Perhaps would be good to include the questionId in the console.log statement for better clarity when debugging

Reply:
![image](https://github.com/user-attachments/assets/de1b3dba-dad8-4ee8-a427-d7833b6e13b8)
now included